massgov/mayflower
# DP-9580 Remove raw fileter from _patterns/03-organisms/by-author/location-listing.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw fileter from _patterns/03-organisms/by-author/location-listing.twig.

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/patternlab/ [feature/DP-9580_organisms_location-listing] grep -r 'location-listing.twig' styleguide/source/
styleguide/source//_patterns/05-pages/location-listing.json:      "path": "@organisms/by-author/location-listing.twig",
styleguide/source//assets/js/modules/locationListing.js:    const rawData = ma.locationListing[i]; // Data object created in @organisms/by-author/location-listing.twig
 ~/MassIT/mayflower/patternlab/ [feature/DP-9580_organisms_location-listing]
```

## Related Issue / Ticket

- [DP-9580](https://jira.mass.gov/browse/DP-9580)


## Steps to Test


1. Go to location listing page and check for any display issue with the listing.  Note:  the filter was used for JSON which is the source data of the listing.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
